### ISSUE TITLE
mRemoteNG crashes in Windows 10 1903 OS Build 18362.356

### ISSUE THREAD

**Issue by afedericojr** [mRemoteNG/mRemoteNG#1592]
## Expected Behavior

Should connect as normal

## Current Behavior

Prompts for login and then freezes with the login dialog still visible after clicking OK.

## Possible Solution

Releated to https://github.com/mRemoteNG/mRemoteNG/issues/1487, which was closed prematurely and incorrectly identified as related to a docking station.
I am using a workstation with 3 monitors. Several versions prior have worked in previous builds of Windows, and immediately broke upon upgrade to the latest build of Windows 10.

## Steps to Reproduce (for bugs)

1. Attempt to connect to a saved connection
2. Enter credentials and click OK

## Context

Application is currently unusable

## Your Environment

* Version used: Previously was using an older version which froze, and upgraded to 1.77.1.27654 without resolve.
* Operating System and version (e.g. Windows 10 1709 x64): Windows 10 1903 OS Build 18362.356

**Comment by farosch** [comment]
Can you please give a proper explaination on what you are trying to do?
- What connection protocol are you using?
- Are you providing any login information (username/password)?
- What system are you trying to connect to?

Please also add the logfile and a dump file. See https://mremoteng.readthedocs.io/en/latest/troubleshooting.html

I can't stress this enough:
If you want me/us to help you *please* take the proper time to file an issue!
You aren't goint into your doctors room screaming "aaaaaaaaaaaargh" expecting getting help, either are you? :P

> Releated to #1487, which was closed prematurely and incorrectly identified as related to a docking station.

Very unlikely, as the OP of #1487 clearly stated: 

> If I unplug my dock, RDP, then reconnect my dock everything works as before.

All users in this issue were using multi-GPU setups and like mentioned in this post the issue was resolved for all users after instlling the latest Windows updates.
You can read more about this in #1343 where one of the developers of DisplayLink (the technology in most of the USB docking station) explained this:

> I believe there could have been a change in Windows around RDP and the Indirect Display Interface API which was rolled out in v1903/19H1. That could be why your application terminates when docked whilst the inbox RDP client works.
> A workaround is to change the 3D settings in GPU (or Windows 10) to use the integrated graphics instead of discrete. Impact is likely to only be dual-GPU/hybrid machines.
> A new build of Windows 10 will fix that issue.
> The OS fix has been rolled out to Windows 10 Insiders build 18932 (20H1 - v2003).
> I'd welcome external confirmation it solves it for you too before we can ask for a fix in a monthly patch for Windows 10 v1903.
> Microsoft approved the fix for multi-GPU machines for deployment to Windows 10 v1903 (19H1).
> It is currently tracking to be an optional update starting rolling out on Tuesday 27th August (overall OS patch, not just that) and then pushed to all machines as part of the Tuesday 10th September cumulative patch and newer.
> Optional updates are manually installed by going to Settings > Updates & Security > Windows Update, and clicking on "Check for updates".
> No driver change necessary.

If the above steps don't resolve your issue, yours definitively is another one ;)

From the looks of it my wild guess would be that your problem might be a duplicate of #853 which is a bug in Windows, that also affects office and various other programs. That bug has been aknowledged by Microsoft and a fix will be included with the release of 19H2. Afterwards, it will get ported to 19H1 and RS5.
